option/option.go

```diff
@@ -76,6 +76,38 @@ var Summary = fuego.OptionSummary
 // Description adds a description to the route.
 var Description = fuego.OptionDescription
 
+// OptionSecurity configures security requirements to the route.
```

Comment:
```suggestion
// Security configures security requirements to the route.
```

You renamed it

Reply:
Please update this. And this should be good to go. 😄